ilc/client/WrapApp.js

```diff
@@ -89,13 +96,16 @@ export default class WrapApp {
         this.#appExtraProps = extraProps;
 
         const {slotName} = appIdToNameAndSlot(props.appId);
-        this.#transactionManager.handlePageTransaction(slotName, slotWillBe.rerendered);
+
+        this.#transactionManager.handlePageTransaction(slotName, slotWillBe.rerendered, this.#wrapperName);
 
         await wrapperCallbacks.unmount(props);
 
         if (this.#appState < APP_STATES.bootstrapped) {
             await appCallbacks.bootstrap(props);
         }
+
+        await this.#bundleLoader.loadCssByAppName(this.#appName);
```

Comment:
I don't think that it should be here... If we need to load CSS on app mount - it should be happening inside `appCallbacks.mount(props)`